Updated securitysuite5.2 to new omb 26 14 requirements (#2231)
cisagov/ScubaGear: PowerShell/ScubaGear/baselines/securitysuite.md

Reviewer: I'd also update this requirement to be a bit more focused, since this isn't about premium audit anymore, just creating audit log retention policies.
```suggestion
- The creation of a custom audit
log retention policy requires E5/G5 licenses or E3/G3 licenses with
add-on compliance licenses.
```

Reviewer: I'd recommend cutting this point entirely since retaining logs for longer than a year was a requirement of OMB M-21-31 but not OMB M-26-14.
```suggestion
```

Reviewer: I'd follow-up on the note above with details about the specific licenses required:
```suggestion
- To maintain logs in M365 for longer than 180 days, the user generating the logs must must have an Office 365 E5 or Microsoft 365 E5 license or a Microsoft Purview Suite (formerly known as Microsoft 365 E5 Compliance) or E5 eDiscovery and Audit add-on license.
```

    - [T1562.008: Disable or Modify Cloud Logs](https://attack.mitre.org/techniques/T1562/008/)

#### MS.SECURITYSUITE.5.2v1
Audit logs SHALL be retained and searchable for a minimum of 3 months and retrievable for a minimum of 12 months.

[![BOD 25-01 Requirement](https://img.shields.io/badge/BOD_25--01_Requirement-C41230)](https://www.cisa.gov/news-events/directives/bod-25-01-implementation-guidance-implementing-secure-practices-cloud-services)
[![Automated Check](https://img.shields.io/badge/Automated_Check-5E9732)](#key-terminology)

<!--Policy: MS.SECURITYSUITE.5.2v1; Criticality: SHALL -->
- _Rationale:_ Audit logs may no longer be available when needed if they are not retained for a sufficient time. Increased log retention time gives an agency the necessary visibility to investigate incidents that occurred some time ago.
- _Last modified:_ June 2026

> - _Note:_ If the user generating the audit log doesn't meet the licensing requirements, data is retained according to the highest priority retention policy. This retention might be either the default retention policy for the user's license or the highest priority policy that matches the user and its record type.

Reviewer: I'd recommend adding a bit contextualize this note some more. Something like this:

```suggestion
- _Note:_ To maintain logs in M365 for longer than 180 days, the user generating the logs must meet the license requirement described below. If the user does not meet these requirements, their data is retained according to the highest priority retention policy. This retention might be either the default retention policy for the user's license or the highest priority policy that matches the user and its record type.
```

> - _Note:_ To maintain logs in M365 for longer than 180 days, the user generating the logs must meet the license requirement described below. If the user does not meet these requirements, their data is retained according to the highest priority retention policy. This retention might be either the default retention policy for the user's license or the highest priority policy that matches the user and its record type.

Reviewer 2: Should we specify what the default retention policy is? e.g. "This retention might be either the default retention policy (6 months) for the user's license or the highest priority policy that matches the user and its record type."
